cmd/helm/reset.go

```diff
@@ -32,7 +32,7 @@ import (
 )
 
 const resetDesc = `
-This command uninstalls Tiller (the helm server side component) from your
+This command uninstalls Tiller (the helm server-side component) from your
```

Comment:
- [ ] While we're still pursuing these corrections, why is "helm" written in lowercase here?